apps/quickstart-integrations-monday-code/src/middleware.js

```diff
@@ -12,7 +12,17 @@ export const authorizeRequest = (req, res, next) => {
     if (!authorization && req.query) {
       authorization = req.query.token;
     }
+    if (typeof authorization !== "string") {
+      logger.info("No credentials in request");
+      res.status(401).json({ error: "Not authenticated, no credentials in request" });
+      return;
+    }
     const signingSecret = getSecret(MONDAY_SIGNING_SECRET);
+    if (signingSecret === undefined) {
+      logger.error("Missing MONDAY_SIGNING_SECRET");
+      res.status(500).json({error: "Missing MONDAY_SIGNING_SECRET"});
+      return:
```

Comment:
Same as above, remove the empty return (there's a typo here too with the colon character)
```suggestion

```

Reply:
same here :)
https://github.com/mondaycom/welcome-apps/blame/0fe9338fc56a145629bbb4de07cbc9865702cb1b/apps/quickstart-integrations-ts/src/middlewares/authentication.ts#L35